lib/typeprof/core/ast/control.rb

```diff
@@ -392,11 +392,75 @@ def undefine0(genv)
 
       def install0(genv)
         ret = Vertex.new(self)
+
+        vars = []
+        @body.modified_vars(@lenv.locals.keys, vars) if @body
+        @rescue_clauses.each do |clause|
+          clause.modified_vars(@lenv.locals.keys, vars)
+        end
+        @else_clause.modified_vars(@lenv.locals.keys, vars) if @else_clause
+        @ensure_clause.modified_vars(@lenv.locals.keys, vars) if @ensure_clause
```

Comment:
```suggestion
```

Are these lines necessary? I tried deleting them, but the test results did not seem to change.
If you have a test case where these would be needed, it would be helpful if you could indicate it.

Reply:
It was a lack of understanding. It was unnecessary and I deleted it.